lib/internal/modules/esm/resolve.js

```diff
@@ -1012,6 +1021,9 @@ function defaultResolve(specifier, context = {}, defaultResolveUnused) {
         return { url: href };
       }
       if (missing) {
+        // Prevent network requests from firing if resolution would be banned.
+        // Network requests can extract data by doing things like putting
+        // secrets in query params
```

Comment:
I put it here because this was the line where someone commented asking for clarification about it. Both places throw the same `ERR_MANIFEST_DEPENDENCY_MISSING`.

I could put it at the bottom and use a footnote-style comment `// [1]`